src/pages/settings/Payments/AddPayPalMePage.js

```diff
@@ -22,107 +22,90 @@ import Icon from '../../../components/Icon';
 import * as Expensicons from '../../../components/Icon/Expensicons';
 import variables from '../../../styles/variables';
 
-class AddPayPalMePage extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            payPalMeUsername: '',
-            payPalMeUsernameError: false,
-        };
-        this.setPayPalMeUsername = this.setPayPalMeUsername.bind(this);
-        this.focusPayPalMeInput = this.focusPayPalMeInput.bind(this);
-    }
+const AddPayPalMePage = (props) => {
+    const [payPalMeUsername, setPayPalMeUsername] = useState('');
+    const [payPalMeUsernameError, setPayPalMeUsernameError] = useState(false);
+    const payPalMeInput = useRef(null);
 
     /**
-     * Sets the payPalMeUsername for the current user
+     * Sets the payPalMe username and error data for the current user
      */
-    setPayPalMeUsername() {
-        const isValid = ValidationUtils.isValidPaypalUsername(this.state.payPalMeUsername);
-        if (!isValid) {
-            this.setState({payPalMeUsernameError: true});
+    const setPayPalMeData = () => {
+        if (!ValidationUtils.isValidPaypalUsername(payPalMeUsername)) {
+            setPayPalMeUsernameError(true);
             return;
         }
-        this.setState({payPalMeUsernameError: false});
-        User.addPaypalMeAddress(this.state.payPalMeUsername);
+        setPayPalMeUsernameError(false);
+        User.addPaypalMeAddress(payPalMeUsername);
 
-        Growl.show(this.props.translate('addPayPalMePage.growlMessageOnSave'), CONST.GROWL.SUCCESS, 3000);
+        Growl.show(props.translate('addPayPalMePage.growlMessageOnSave'), CONST.GROWL.SUCCESS, 3000);
         Navigation.navigate(ROUTES.SETTINGS_PAYMENTS);
-    }
-
-    focusPayPalMeInput() {
-        if (!this.payPalMeInputRef) {
-            return;
-        }
-
-        this.payPalMeInputRef.focus();
-    }
+    };
```

Comment:
I think we should be using `useCallback()` here as per [this pending style guide advice](https://github.com/Expensify/App/pull/16882/files#diff-2f2230dab0f5046b9596723a603e0a31478df2a75a88afd3d9cbb6a331e2bcbdR627). Basically, we will keep recreating this `setPayPalMeData()` function on each render and since it has dependencies then we can use `useCallback()`.